bitnami/cassandra/templates/statefulset.yaml

```diff
@@ -88,6 +99,9 @@ spec:
               {{ .Values.entrypoint }} {{ .Values.cmd }}
```

Comment:
For standardizing, I would use the `command` and `args` section that we use in others. It is a breaking change after all.

Reply:
Also, I believe that we can hardcode the `{{ .Values.entrypoint}} {{ .Values.cmd }}`. This comes from the time we were using rolling tags and we transitioned from node.js logic to bash logic